sources/org.osbuild.files

```diff
@@ -44,8 +46,18 @@ def fetch(url, checksum, directory):
                 "--fail",
                 "--location",
                 "--output", checksum,
-                url
-            ], encoding="utf-8", cwd=tmpdir, check=False)
+            ]
+            if secrets is not None:
```

Comment:
You don't really need the explicit `is not None` check here (and everywhere), just `if secrets:` is fine.

Reply:
These are removed